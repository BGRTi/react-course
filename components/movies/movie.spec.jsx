import React from 'react';
import { shallow } from 'enzyme';

import MoviesContainer from './movies';
import CardsList from '../cards-list/cards-list';

function setup() {
  const wrapper = shallow(<MoviesContainer />);
  return { wrapper };
}

describe('MoviesContainer Test Suite', () => {
  it('render a MoviesContainer', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });
  it('Should have an image', () => {
    const { wrapper } = setup();
    expect(wrapper.find(CardsList).exists()).toBe(true);
  });
});
