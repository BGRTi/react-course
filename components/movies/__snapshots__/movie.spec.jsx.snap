// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesContainer Test Suite render a MoviesContainer 1`] = `ShallowWrapper {}`;
